cdap-docs/admin-manual/source/_includes/installation/configuration.txt

```diff
@@ -162,28 +162,56 @@ Guide <https://hbase.apache.org/book.html#ulimit>`__.
 
 .. highlight:: console
 
+.. _|distribution|-configuration-local-storage:
 .. _|distribution|-configuration-tmp-files:
 
-Writing to Temp Files
----------------------
-Temp directories, depending on the distribution, are utilized by CDAP (the first two of
-this set of directories are properties specified in the ``cdap-site.xml`` file, as
-described in the :ref:`appendix-cdap-site.xml`):
-
-- ``app.temp.dir`` (default: ``/tmp``)
-- ``kafka.log.dir`` (default: ``/tmp/kafka-logs``)
-- ``/var/cdap/run``
-- ``/var/log/cdap``
-- ``/var/run/cdap``
-- ``/var/tmp/cdap`` 
-
-The CDAP user (the ``cdap`` system user) **must** be able to write to these directories, as they
-are used for deploying applications and for operating CDAP.
-
-As in all installations, the ``kafka.log.dir`` may need to be created locally. If you
-configure ``kafka.log.dir`` (or any of the other settable parameters) to a particular
-directory, you need to make sure that **the directory exists** and that it **is writable**
-by the CDAP user.
+Local Storage Configuration
+---------------------------
+Local storage and temp directories |---| depending on the distribution |---| are utilized
+by CDAP for deploying applications and operating CDAP.
+
+The CDAP user (the ``cdap`` system user) **must** be able to write to **all** of these
+directories, as they are used for deploying applications and for operating CDAP.
+  
+- List of local storage and temp directories:
+
+  - Properties specified in the ``cdap-site.xml`` file, as described in the :ref:`appendix-cdap-site.xml`:
+
+    - ``app.temp.dir`` (default: ``/tmp``)
+    - ``kafka.server.log.dirs`` (default: ``/tmp/kafka-logs``)
+    - ``local.data.dir`` (default: ``data``; if this is instead an absolute path, needs to be writable)
+
+  - Temp directories:
```

Comment:
i'd suggest renaming this since these directories aren't actually temporary.  For reference, this should all get cleaned up for 4.0, but:

/var/cdap/run: created by the packages, used as pid dir
/var/log/cdap: created by the packages, used as log dir
/var/cdap/run: default cdap user's home, created by the packages
/var/tmp/cdap: default local_dir, defined & created in init scripts